packages/protocol/contracts/common/GasPriceMinimum.sol

```diff
@@ -1,43 +1,46 @@
 pragma solidity ^0.5.8;
 
+import "fixidity/contracts/FixidityLib.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "./Initializable.sol";
 import "./UsingRegistry.sol";
+import "./UsingFixidity.sol";
 import "../stability/interfaces/ISortedOracles.sol";
-import "../common/FractionUtil.sol";
 
 /**
  * @title Stores and provides gas price minimum for various currencies.
  */
-contract GasPriceMinimum is Ownable, Initializable, UsingRegistry {
+contract GasPriceMinimum is Ownable, Initializable, UsingRegistry, UsingFixidity {
+  using FixidityLib for int256;
   using SafeMath for uint256;
-  using FractionUtil for FractionUtil.Fraction;
 
   event TargetDensitySet(
-    uint256 numerator,
-    uint256 denominator
+    int256 targetDensity
```

Comment:
Hmm, this won't be very readable, should we still emit a numerator and denominator? Ditto for everything below

Reply:
The denominator would be the same in every case. How would you feel about naming this and similar event arguments `targetDensityFixed`?

Reply:
I think it's a similar case to ERC20 balance. Where numbers only make sense after you query `decimals()`. So, i don't mind this change actually.

Reply:
maybe expose a `digits()` || `decimals()` here